service/matching/tasklist/matcher.go

```diff
@@ -26,8 +26,7 @@ import (
 	"fmt"
 	"time"
 
-	"golang.org/x/time/rate"
-
+	"github.com/uber/cadence/common/clock"
```

Comment:
changes LGTM. do we have integration test covering `MustOffer` (not necessarily how much it waited etc.)?

Reply:
not entirely sure, but good point 🤔 

coverage should tell us, or I'll see if I can figure out / find someone who knows our integration tests and if they might hit this.

Reply:
I think the unit tests Zijian was looking at cover that

Reply:
No unit tests covering how much it waited, but locally I do get coverage on the ratelimit branches for MustOffer, and some coverage runs show it too: https://coveralls.io/builds/68006141/source?filename=service%2Fmatching%2Ftasklist%2Fmatcher.go#L217
so that is exercised to some degree, but I'm not sure exactly how much.